src/Disks/IO/ThreadPoolReader.cpp

```diff
@@ -36,6 +36,8 @@
         #define SYS_preadv2 380
     #elif defined(__riscv)
         #define SYS_preadv2 286
+    #elif defined(__loongarc64)
```

Comment:
`__loongarc64`? Should that not be `__loongarch64`?

Reply:
That should be `loongarch64`, it's my mistake.